x/axelarnet/types/genesis.go

```diff
@@ -29,7 +29,7 @@ func DefaultGenesisState() *GenesisState {
 			AddrPrefix: "axelar",
 			Assets: []Asset{{
 				Denom:     "uaxl",
-				MinAmount: sdk.NewInt(1000000),
+				MinAmount: sdk.NewInt(100000),
```

Comment:
maybe we should actually make it 1000uaxl? @sergeynog any thoughts?

Reply:
That'll be too low since our gas fees will be around 0.05AXL